components/Tests/TestEditorBuilder.tsx

```diff
@@ -0,0 +1,42 @@
+﻿import React from "react";
```

Comment:
like before - I think we can get rid of this whole file, as the functions aren't really doing anything